wpimath/src/main/java/edu/wpi/first/math/controller/SimpleMotorFeedforward.java

```diff
@@ -31,12 +37,34 @@ public class SimpleMotorFeedforward implements ProtobufSerializable, StructSeria
   private final double m_dt;
 
   /**
-   * Creates a new SimpleMotorFeedforward with the specified gains and period. Units of the gain
+   * Create a new SimpleMotorFeedforward with the specified plant. Units used to
+   * create the plant
+   * will dictate units of the computed feedforward. The static gain (ks) is
+   * assumed to be 0.0.
+   * 
+   * <p>
+   * The constructor is useful for simulating LinearSystemSim classes.
+   * 
+   * @param plant     The system plant
+   * @param dtSeconds The period in seconds.
+   * @throws IllegalArgumentException for period &le; zero.
+   */
+  public SimpleMotorFeedforward(LinearSystem<N2, N1, N2> plant, double dtSeconds) {
```

Comment:
Should be 1x1x1 because there's only a velocity state.

Reply:
I see now.  So I would have to slide a position plant passing it into the constructor?

Reply:
If you made one, yea. There's also a LinearSystemId function for flywheels that produces a 1x1 system.

Reply:
I gotcha.  So the elevator and arm wrappers would need the 2x1x2 plant since that's the only way to physically create it in sysId.